addons/rose/addon/utilities/component-auto-id.js

```diff
@@ -3,12 +3,12 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { v1 } from 'ember-uuid';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
In this specific case, I decided to go with `v4`, but `uuid` has `v1` available in case we feel we want to respect the uuid v generated before.

Reply:
I would also just use `v4`